src/components/organisms/AuthenticationModal/RegisterForm/RegisterForm.tsx

```diff
@@ -92,35 +92,34 @@ const RegisterForm: React.FunctionComponent<PropsType> = ({
       setShowLoginModal(false);
       const auth = await signUp(data);
 
-      if (venue.access?.includes(VenueAccessMode.Codes)) {
-        const result = await checkAccess({
-          venueId: venue.id,
-          code: data.code,
-        });
-        console.log("result.data", result.data);
-        console.log(
-          "saving token to",
-          accessTokenKey(venue.id),
-          ": ",
-          result.data.token
-        );
-        setLocalStorageToken(venue.id, result.data.token);
+      let result = null;
+
+      switch (venue.access) {
+        case VenueAccessMode.Codes:
+          result = await checkAccess({
+            venueId: venue.id,
+            code: data.code,
+          });
+          break;
+        case VenueAccessMode.Emails:
+          result = await checkAccess({
+            venueId: venue.id,
+            email: data.email,
+          });
+          break;
+
+        default:
+          break;
       }
```

Comment:
As called out in `src/components/organisms/AuthenticationModal/LoginForm/LoginForm.tsx` (see comments in this review), this `switch` statement doesn't seem to be needed, and we can simplify it right down.

I believe my comments in a previous review also called out that file and this sharing the exact same code, and asking for it to be refactored in a way that they both call a common helper, rather than it being copy/paste repeated here. Since that seems like it got missed, I'm requesting it here again this time around.